core/src/main/java/org/apache/calcite/rel/type/RelDataTypePrecedenceList.java

```diff
@@ -31,7 +31,7 @@ public interface RelDataTypePrecedenceList {
    * Determines whether a type appears in this precedence list.
    *
    * @param type type to check
-   * @return true iff this list contains type
+   * @return true if this list contains type
```

Comment:
iff means "if and only if"
was the original comment wrong?

Reply:
I misunderstood. I will rollback this.